dropwizard-jetty/src/main/java/io/dropwizard/jetty/HttpConnectorFactory.java

```diff
@@ -650,16 +651,12 @@ protected ServerConnector buildConnector(Server server,
         connector.setPort(port);
         connector.setHost(bindHost);
         connector.setInheritChannel(inheritChannel);
-        if (acceptQueueSize != null) {
-            connector.setAcceptQueueSize(acceptQueueSize);
-        } else {
-            // if we do not set the acceptQueueSize, when jetty
-            // creates the ServerSocket, it uses the default backlog of 50, and
-            // not the value from the OS.  Therefore we set to the value
-            // obtained from NetUtil, which will attempt to read the value from the OS.
-            // somaxconn setting
-            connector.setAcceptQueueSize(NetUtil.getTcpBacklog());
-        }
+        // if we do not set the acceptQueueSize, when jetty
+        // creates the ServerSocket, it uses the default backlog of 50, and
+        // not the value from the OS.  Therefore we set to the value
+        // obtained from NetUtil, which will attempt to read the value from the OS.
+        // somaxconn setting
+        connector.setAcceptQueueSize(Objects.requireNonNullElseGet(acceptQueueSize, NetUtil::getTcpBacklog));
```

Comment:
Could we please extract the actual accept queue size into a variable?

Reply:
Sure - to give something to look at when debugging?

Reply:
> to give something to look at when debugging?

Yes, exactly.